### ISSUE TITLE
Linux Guest - unable to compile STAP kernel extension

### ISSUE THREAD

**Issue by enceladus3** [kevoreilly/CAPEv2#567]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [y] I am running the latest version
- [y] I checked the documentation and found no answer
- [y] I checked to make sure that this issue has not already been filed
- [y] I'm reporting the issue to the correct repository (for multi-repository projects)
- [y] I'm have read all configs with all optional parts

# Expected Behavior

Have a stap_.ko file

# Current Behavior

no stap_.ko and unable to obtain it due to errors

# Failure Information (for bugs)

## Steps to Reproduce

Configure a Linux guest as it is presented in the guide but for 18.04: https://github.com/kevoreilly/CAPEv2/blob/master/docs/book/src/installation/guest/linux.rst

## Context

Trying to setup a guest. Already tried with Ubuntu  17.10 but many steps from the guide failed due to various reasons (including the distro being EOL).
Moved forward and tried the entire setup on 18.04. I was able to install kernel debugging symbols and also successfully patch the SystemTap tapset, so that the CAPE analyzer can properly parse the output using:

```
wget https://raw.githubusercontent.com/kevoreilly/CAPEv2/master/extra/systemtap/expand_execve_envp.patch

wget https://raw.githubusercontent.com/kevoreilly/CAPEv2/master/extra/systemtap/escape_delimiters.patch

sudo patch /usr/share/systemtap/tapset/linux/sysc_execve.stp < expand_execve_envp.patch

sudo patch /usr/share/systemtap/tapset/uconversions.stp < escape_delimiters.patch
```

and this wiki from Ubuntu:

https://wiki.ubuntu.com/Debug%20Symbol%20Packages

However I encountered issues while running:

`sudo stap -p4 -r $(uname -r) strace.stp -m stap_ -v
`
Please see the logs file attached below. It also contains some of the commands I used so that you can see the patch is successfully applied.
At line 46 I tried to compile it.

| Question         | Answer
|------------------|--------------------
| OS version       | Ubuntu 18.04 (Guest) since 17.10 is EOL

## Failure Logs
[stre_stp.txt](https://github.com/kevoreilly/CAPEv2/files/7047894/stre_stp.txt)

**Comment by doomedraven** [comment]
if you see your log you have tons of errors and it not compile so you can't run what isn't compiled

**Comment by enceladus3** [comment]
Hi @doomedraven,

Indeed. This time, however, I was able to successfully apply the patches so I thought I might have a different case.

Ok, so the only way to get it working on something different than 17.10 is to edit this, right?https://github.com/kevoreilly/CAPEv2/blob/1e66d2460276b28b45bea8123cc74daa83295f68/extra/systemtap/expand_execve_envp.patch

Jsut to make it clear, 17.10 is still supported? Should I invest time in making it work on thins distro? I can contribute the project since I experimented with 17.10 and I have something to add.

Thanks
